relayer/src/chain/cosmos.rs

```diff
@@ -2371,8 +2458,10 @@ fn client_id_suffix(client_id: &ClientId) -> Option<u64> {
 pub struct TxSyncResult {
     // the broadcast_tx_sync response
     response: Response,
-    // the events generated by a Tx once executed
+    // the IBC events generated by a Tx once executed
     events: Vec<IbcEvent>,
```

Comment:
Now that there's a `cosmos_events` field, this field should also probably be named more explicitly. 
```suggestion
    ibc_events: Vec<IbcEvent>,
```